vite.config.ts

```diff
@@ -3,27 +3,37 @@ import packageJson from "./package.json";
 import react from "@vitejs/plugin-react";
 
 export default defineConfig({
-  plugins: [react()],
+  plugins: [
+    react({
+      jsxImportSource: "@emotion/react",
+      babel: {
+        plugins: ["@emotion/babel-plugin"]
+      }
+    })
+  ],
+  esbuild: {
+    logOverride: { "this-is-undefined-in-esm": "silent" }
```

Comment:
Sadly necessary to ensure that our jsx transforms don't cause warnings in our output.